java-checks/src/test/files/checks/AssertionsInTestsCheckJunit4.java

```diff
@@ -0,0 +1,49 @@
+import javax.annotation.Nullable;
+import org.junit.Test;
+
+public class AssertionsInTestsCheckTest {
```

Comment:
No big deal but, why two files ? given how the check is written, two classes in one file should be sufficient.